src/syft/proto/grid/service/signaling_service_pb2.py

```diff
@@ -2,6 +2,7 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: proto/grid/service/signaling_service.proto
 """Generated protocol buffer code."""
+# third party
```

Comment:
I don't know if it's a big deal or not but maybe we could avoid formatting generated files?

Reply:
We are already modifying these files to change the pb2 imports, by formatting them with the same formatter we use on all our other python code we avoid the issue of having to add "blocklist" files to our formatting systems which need to be maintained. Since we have the formatting tool and rules defined as part of the pre-commit hooks and CI these should always be the same for everyone so there will be 0 noise.

Happy to look at removing it but this actually seemed easier and without consequence.